Reply/app/src/main/res/layout/email_item_layout.xml

```diff
@@ -31,30 +31,40 @@
         android:id="@+id/email_frame"
         android:layout_height="wrap_content"
         android:layout_width="match_parent"
+        android:layout_marginEnd="@dimen/grid_2"
+        android:layout_marginStart="@dimen/grid_2"
         android:layout_marginVertical="@dimen/grid_0_25"
-        android:layout_marginHorizontal="@dimen/grid_0_5"
-        android:background="?attr/colorSecondary">
+        android:layout_marginHorizontal="@dimen/grid_0_5">
 
         <com.google.android.material.card.MaterialCardView
             android:id="@+id/card_view"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:elevation="@dimen/plane_00"
             android:clickable="true"
             android:focusable="true"
             android:onClick="@{(view) -> listener.onEmailClicked(view, email)}"
             android:onLongClick="@{(view) -> listener.onEmailLongPressed(email)}"
             app:cardPreventCornerOverlap="false"
+            style="?attr/materialCardViewElevatedStyle"
```

Comment:
since the mocks don't have any shadows on the cards, i'm wondering if we should use the materialCardViewFilledStyle instead. can you try that out and attach a new screenshot so we can see how it looks?

if you look closely in the screenshot, you can actually see that the cards have an elevation shadow that is getting cut off:

![image](https://user-images.githubusercontent.com/1420597/136439535-df44d6ec-01d3-42e5-be9f-e4962d2fb6f6.png)